source/shared/cpp/ObjectModel/ParseContext.cpp

```diff
@@ -261,12 +263,18 @@ namespace AdaptiveSharedNamespace
     void ParseContext::SaveContextForCollectionTypeElement(const std::shared_ptr<CollectionTypeElement>& current)
     {
         // save current style value
-        m_parentalContainerStyles.push_back(current->GetStyle());
+        if (current->GetStyle() != ContainerStyle::None)
```

Comment:
>None [](start = 51, length = 4)

Should this be `None` or `NotSet`?

Reply:
None is correct. container will have a None style.

---
In reply to: [266114732](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266114732) [](ancestors = 266114732)

Reply:
okay, sounds good :)

---
In reply to: [266114969](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r266114969) [](ancestors = 266114969,266114732)